.changeset/great-feet-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: reject unknown pseudo-classes even if their names match with existing pseudo-elements
```

Comment:
[suggest] Thanks for creating the changelog. Can you update it for consistency? See also our [convention](https://stylelint.io/maintainer-guide/pull-requests#merging).

```suggestion
Fixed: `selector-pseudo-class-no-unknown` false negatives for non-standard `:slotted` and `:placeholder`
```

Note: I've updated the PR title.

Reply:
"for non-standard `:slotted` and `:placeholder`" is too specific
Ill try to come up with something better

Reply:
Make sense. I've updated the PR title also.